lua/lazyvim/plugins/extras/lang/tex.lua

```diff
@@ -0,0 +1,61 @@
+return {
+  {
+    "folke/which-key.nvim",
+    opts = {
+      defaults = {
+        ["<LocalLeader>l"] = { name = "+vimtex" },
+      },
+    },
+  },
+
+  -- Add BibTeX/LaTeX to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "bibtex", "latex" })
+      end
+      if type(opts.highlight.disable) == "table" then
+        vim.list_extend(opts.highlight.disable, { "latex" })
+      else
+        opts.highlight.disable = { "latex" }
+      end
+    end,
+  },
+
+  {
+    "lervag/vimtex",
+    version = "*",
+    ft = { "bib", "tex" },
```

Comment:
Do not lazy-load VimTeX. That is not supported. (It needs to set things up _before_ you set the filetype.)

Reply:
lazy-loading works fine though last time I checked, but definitely something OP should verify/test to be 100% sure

Reply:
Did you test inverse search (which is a core feature)?

From the docs: 
> Note: Many plugin managers provide mechanisms to lazy load plugins. There is
      no need to use such a mechanism for VimTeX, and in fact, doing it will
      prevent Vim/neovim from loading the :VimtexInverseSearch command.

I'd respect the dev's intent here.

Reply:
Also, VimTeX comes with improved `ftdetect` scripts, which again are broken by lazy-loading.

Reply:
ok, great